app/services/poem_creation_service.rb

```diff
@@ -0,0 +1,39 @@
+class PoemCreationService
+  class << self
+    def create(user, parameters, view_context: nil)
+      new.create(user, parameters, view_context: view_context)
+    end
+  end
+
+  def create(user, parameters, view_context: nil)
+    poem = user.poems.build(parameters)
+    if poem.save
```

Comment:
__[rubocop]__
* Favor modifier `if` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`. [:ref](https://github.com/bbatsov/ruby-style-guide#if-as-a-modifier)